Hearthstone Deck Tracker/Utility/WotogCounterHelper.cs

```diff
@@ -24,6 +25,16 @@ public static class WotogCounterHelper
 		public static bool? YoggInDeck => DeckContains(CardIds.Collectible.Neutral.YoggSaronHopesEnd);
 		public static bool? ArcaneGiantInDeck => DeckContains(CardIds.Collectible.Neutral.ArcaneGiant);
 
+		public static int PlayerMaxJadeGolem => Core.Game.Player.PlayerEntities.
+					Where(x => IsJadeGolem(x) && !x.Info.Stolen).
+					Max(x => (Nullable<Int32>)x.Card.Attack) ?? 0;
```

Comment:
Theres is a `JADE_GOLEM` tag on the player entity (`Core.Game.PlayerEntity` and `OpponentEntity` respectively), which is current max `ATK`/`HEALTH`. You should use that one.